code/datums/uplink/stealthy and inconspicuous weapons.dm

```diff
@@ -24,8 +24,13 @@
 	item_cost = 4
 	path = /obj/item/cartridge/syndicate
 
+/datum/uplink_item/item/stealthy_weapons/pens
+	name = "Special Pens Kit"
```

Comment:
I feel like this could use a name that more clearly represents what the pens do. "Special Pens" sounds like pens that write in different fonts.

Reply:
It would be best if it had description. What other name you suggest?

Reply:
"Pen-jector Kit" with "(Healing, Pacifying, Hyperzine, Poison)" behind it, or some such.